package.json

```diff
@@ -70,14 +72,23 @@
     "@storybook/manager-webpack5": "^6.5.15",
     "@storybook/react": "^6.5.15",
     "@storybook/testing-library": "^0.0.13",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
     "@types/node": "^18.11.13",
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
+    "@types/testing-library__jest-dom": "^5.14.5",
+    "babel-jest": "^29.3.1",
```

Comment:
maybe it's worth switching to ts-jest in a new project. we won't need `babelTransform.js`

Reply:
I'm not a fan of typechecking while running tests. It's way way too slow

Reply:
okay ts-jest with isolatedModules looks equally fast